test/unit/collection-view.spec.js

```diff
@@ -28,6 +28,34 @@ describe('collection view', function() {
   // Collection View Specs
   // ---------------------
 
+  describe('when a collection view is DOM', function() {
+    beforeEach(function() {
+      this.$fixtureEl = $('<div id="fixture-collectionview"></div>');
```

Comment:
can we change the fixture to cater for when childViews are pre-rendered as well?